### PR TITLE
:memo: Updated docs and fixed docs,variable name,function names typos

### PR DESCRIPTION
**Pull request by Ujas-Majithiya**
Also removed dates from changelog as they are present in Versions tab of pub.dev

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -291,13 +291,14 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
                 ),
               ],
             ),
+            const SizedBox(height: 10),
```
Comment (DhavalRKansara) [inline example/lib/main.dart:294]: Any reason for adding this SizedBox?

File: lib/src/audio_file_waveforms.dart [hunk 2]
```diff
@@ -8,57 +8,64 @@ import '../audio_waveforms.dart';
...
 
-  ///Curve for animation. Defaults to Curves.bounceOut
+  /// Curve for animation. Defaults to Curves.bounceOut
   final Curve animationCurve;
 
-  ///Density of the display. Providing accurate density is not neccesary, if desired looking
-  ///waveforms are needed.
+  /// Density of the display. Providing accurate density is not necessary,
+  /// if desired looking waveforms are needed.
   ///
```
Comment (DhavalRKansara) [inline lib/src/audio_file_waveforms.dart:44]: remove this extra comment lines

File: lib/src/base/wave_style.dart [hunk 3]
```diff
@@ -1,77 +1,90 @@
...
+  /// Use as below,
+  ///
+  /// import 'dart:ui' as ui show Gradient;
+  ///
+  /// ...
+  ///
+  /// ui.Gradient.linear(
+  ///                 const Offset(70, 50),
+  ///                 Offset(MediaQuery.of(context).size.width / 2, 0),
+  ///                 [Colors.red, Colors.green],
```
Comment (DhavalRKansara) [inline lib/src/base/wave_style.dart:79]: any reason for living commented code?

**Review by DhavalRKansara — CHANGES_REQUESTED** [review]

File: lib/src/base/wave_style.dart (hunk 3 above)
Comment (ujas-m-simformsolutions) [reply lib/src/base/wave_style.dart]: This is an example code snippet for showing how to use shader while applying gradient

File: lib/src/audio_file_waveforms.dart (hunk 2 above)
Comment (ujas-m-simformsolutions) [reply lib/src/audio_file_waveforms.dart:44]: It provides new line for next comment so it's required

File: example/lib/main.dart (hunk 1 above)
Comment (ujas-m-simformsolutions) [reply example/lib/main.dart]: Updated with safearea

**Pull request merged**